backend/band-assignments/gradlew.bat

```diff
@@ -0,0 +1,91 @@
+@rem
+@rem Copyright 2015 the original author or authors.
+@rem
+@rem Licensed under the Apache License, Version 2.0 (the "License");

```

Comment:
I am not familiar with this License; do we need to cite it in the readme?

Reply:
OK I looked it up:
"...indicates the license under which the file is distributed. Specifically, it states that the file is licensed under the Apache License, Version 2.0.

The Apache License, Version 2.0 is an open-source license that allows you to use, modify, and distribute the software covered by the license. It is a widely used license in the open-source community."

We could include it in the README, or a separate License file.